src/kdbindings/signal.h

```diff
@@ -178,11 +194,12 @@ class Signal
         friend class Signal;
         struct Connection {
             std::function<void(Args...)> slot;
-            bool isDeferred{ false };
+            std::function<void(const ConnectionHandle &, Args...)> slotDeferred;
```

Comment:
Ideally this would be a `union`, as adding a second `std::function` is quite expensive (`sizeof(std::function) ~~ 32 byte`), but I guess it's good enough for now.